content/source/docs/enterprise/api/oauth-tokens.html.md

```diff
@@ -12,50 +12,218 @@ The `oauth-token` object represents a VCS configuration which includes the OAuth
 
 ## List OAuth Tokens
 
-List all the OAuth Tokens for a given organization
+List all the OAuth Tokens for a given OAuth Client
 
-| Method | Path           |
-| :----- | :------------- |
-| GET | /organizations/:organization_name/oauth-tokens |
+`GET /oauth-clients/:oauth_client_id/oauth-tokens`
 
-### Parameters
+Parameter            | Description
+---------------------|------------
+`:oauth_client_id`   | The ID of the OAuth Client
 
-- `:organization_name` (`string: <required>`) - specifies the organization name where the OAuth Token is defined
+Status  | Response                                     | Reason
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "oauth-tokens"`) | Success
+[404][] | [JSON API error object][]                    | OAuth Client not found, or user unauthorized to perform action
 
 ### Sample Request
 
 ```shell
 curl \
   --header "Authorization: Bearer $TOKEN" \
-  https://app.terraform.io/api/v2/organizations/my-organization/oauth-tokens
+  https://app.terraform.io/api/v2/oauth-clients/oc-GhHqb5rkeK19mLB8/oauth-tokens
 ```
 
 ### Sample Response
 
 ```json
 {
-  "data": [
-    {
-      "id":"ot-hmAyP66qk2AMVdbJ",
-      "type":"oauth-tokens",
-      "attributes": {
-        "uid":"885724",
-        "created-at":"2017-11-02T06:37:49.284Z",
-        "service-provider-user":"skierkowski",
-        "has-ssh-key":false
-      },
-      "relationships": {
-        "oauth-client": {
-          "data": {
-            "id":"oc-GhHqb5rkeK19mLB8",
-            "type":"oauth-clients"
-          }
+  "data": {
+    "id": "ot-hmAyP66qk2AMVdbJ",
+    "type": "oauth-tokens",
+    "attributes": {
+      "created-at":"2017-11-02T06:37:49.284Z",
+      "service-provider-user":"skierkowski",
+      "has-ssh-key": false
+    },
+    "relationships": {
+      "oauth-client": {
+        "data": {
+          "id": "oc-GhHqb5rkeK19mLB8",
+          "type": "oauth-clients"
+        },
+        "links": {
+          "related": "/api/v2/oauth-clients/oc-GhHqb5rkeK19mLB8"
         }
-      },
-      "links": {
-        "self":"/api/v2/oauth-tokens/ot-hmAyP66qk2AMVdbJ"
       }
+    },
+    "links": {
+      "self": "/api/v2/oauth-tokens/ot-hmAyP66qk2AMVdbJ"
     }
-  ]
+  }
```

Comment:
Did the response body change from an array of objects to a single object? Or was that a copy-paste issue?

Reply:
The response had been copied from another part of the docs. It's been fixed. Thanks!